build.gradle

```diff
@@ -1,13 +1,17 @@
 plugins {
     id 'maven-publish'
+    id 'org.ajoberstar.grgit' version '4.0.2'
```

Comment:
4.1.0 seems to be latest plugin version. https://plugins.gradle.org/plugin/org.ajoberstar.grgit

Reply:
Good spot, upgraded.